twine-cert.txt

```diff
@@ -76,273 +96,353 @@ de = Aktualisieren
+en = Is your destination not listed? More countries are progressively being added in stages. \n\nYou can find more information in the frequently asked questions and on the #Re-open EU::Re-open EU# information page.
 tags = common
 [certificate_check_validity_selection_country_action_button]
 de = Übernehmen
+en = Select
 tags = common
 [certificate_check_validity_selection_date]
 de = Datum
+en = Date
 tags = common
 [certificate_check_validity_selection_date_title]
 de = Datum wählen
+en = Select Date
 tags = common
 [certificate_check_validity_selection_date_action_button]
 de = Weiter
+en = Next
 tags = common
 [certificate_check_validity_selection_time_title]
 de = Zeit wählen
+en = Select time
 tags = common
 [certificate_check_validity_selection_time_action_button]
 de = OK
+en = OK
 tags = common
 [certificate_check_validity_loading]
 de = Zertifikate werden geprüft
+en = Certificates are being checked
 tags = common
 [certificate_check_validity_no_certificate_saved_title]
 de = Sie haben noch keine Zertifikate gespeichert
+en = You have not saved any certificates yet
 tags = common
 [certificate_check_validity_no_certificate_saved_message]
 de = Fügen Sie ein Zertifikat hinzu, um dessen Gültigkeit zu überprüfen.
+en = Add a certificate so you can check its validity.
 tags = common
 [certificate_check_validity_vaccination]
 de = Impfzertifikat
+en = Vaccination certificate 
 tags = common
 [certificate_check_validity_test]
 de = Testzertifikat
+en = Test certificate
 tags = common
 [certificate_check_validity_recovery]
 de = Genesenenzertifikat
+en = Recovery certificate
 tags = common
 [certificate_check_validity_result_valid]
 de = Gültig
+en = Valid
 tags = common
 [certificate_check_validity_result_not_valid]
 de = Im Reiseland ungültig
+en = Invalid in this country
 tags = common
 [certificate_check_validity_result_not_testable]
 de = Nicht vollständig prüfbar
+en = Full check not possible
 tags = common
 [certificate_check_validity_note]
 de = Beachten Sie, dass sich die Einreiseregeln ändern können. Prüfen Sie daher die Zertifikate mindestens 48 Stunden vorher. Es können in einzelnen Regionen oder bei Ihrer Rückkehr weitere Regeln oder Einschränkungen gelten. Zudem ist es möglich, dass weitere Dokumente für eine Einreise notwendig sind.\n\nMehr Informationen finden Sie in den Häufigen Fragen und unter #https://reopen.europa.eu::https://reopen.europa.eu#.
+en = Please be aware that travel regulations can change. You should check your certificates at least 48 hours before you travel. Different regulations or restrictions might apply in some regions or when you return. You might also need additional documents to travel to a different country. \n\nYou can find more information in the frequently asked questions and on the #Re-open EU::Re-open EU# information page. 
```

Comment:
Please use URL like in the German translation

Reply:
Alright. I changed it.